src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/DebugHelper.kt

```diff
@@ -0,0 +1,23 @@
+package org.vaccineimpact.orderlyweb.customConfigTests
+
+import org.junit.rules.TestWatcher
+import org.junit.runner.Description
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.logging.LogType
+
+class DebugHelper: TestWatcher()
+{
+    lateinit var driver: WebDriver
+
+    override fun failed(e: Throwable, description: Description)
```

Comment:
Although I wonder if we could print the throwable as well.

Reply:
That will already be printed to stderr, see e.g. https://buildkite.com/mrc-ide/orderly-web/builds/1649#14e16032-efe6-4496-8e86-f03fe2bf161b/37-4193